src/media/css/app-list.styl

```diff
@@ -117,9 +117,19 @@ section.langpacks > div {
         &:nth-child(2n) {
             border-right: 0;
             padding-right: 0;
+
+            +rtl() {
+                border-right: $border;
+                padding: 20px 50px 20px 0;
```

Comment:
Not a fan of the `ltr()` idea really since we should just use the cascade and we're not overwriting a mountain of rules. If we go too far we'll eventually need to rewrite the rest of the CSS with either an `ltr()` or `rtl()` for consistency. Your call though our task is interesting enough that I won't let styling choices hold up the work.